e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -349,6 +349,50 @@ Scenario('HMCTS admin vacates and re-lists a hearing', async ({I, caseViewPage,
   I.seeInTab(['Further evidence documents 2', 'Hearing'], 'Case management hearing, 1 January 2060');
 });
 
+Scenario('HMCTS admin re-lists previously cancelled hearing', async ({I, caseViewPage, manageHearingsEventPage}) => {
```

Comment:
vv minor but the hearing is not previously cancelled as you are cancelling it as part of the test